pkg/contracts/src/CVStrategy.sol

```diff
@@ -297,26 +300,33 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
         this.withdraw(_member);
     }
 
-    function increasePower(address _member, uint256 _amountToStake) external view returns (uint256) {
+    function increasePower(address _member, uint256 _amountToStake) external returns (uint256) {
         //requireMemberActivatedInStrategies
 
         uint256 pointsToIncrease = 0;
         if (pointSystem == StrategyStruct.PointSystem.Unlimited) {
             pointsToIncrease = increasePowerUnlimited(_amountToStake);
         } else if (pointSystem == StrategyStruct.PointSystem.Capped) {
             pointsToIncrease = increasePowerCapped(_member, _amountToStake);
+        } else if (pointSystem == StrategyStruct.PointSystem.Quadratic) {
+            pointsToIncrease = increasePowerQuadratic(_member, _amountToStake);
         }
+        totalPointsActivated = pointsToIncrease;
```

Comment:
totalPointsActivated should be summed? And also that its going to the event.

Reply:
i didn't implemented those changes just the events. This is appearing here because i needed to merge the other branch into mine. But yeah seems that the sum is missing